src/types/Users.ts

```diff
@@ -0,0 +1,6 @@
+export interface Users {
```

Comment:
Consider renaming the interface to 'User' instead of 'Users', as it represents a single user object.